docker/Dockerfile

```
@@ -36,7 +36,8 @@ RUN pip install --prefix="/install" --no-warn-script-location \
         frozendict \
         jaeger-client \
         opentracing \
-        prometheus-client \
+        # Match the version constraints of Synapse
+        "prometheus-client<0.9.0" \
```

If we're going to match it, we should match it exactly:

```suggestion
        "prometheus_client>=0.4.0,<0.9.0" \
```